Sources/Sham/Protocols/LaunchEnvironmentObject.swift

```diff
@@ -0,0 +1,11 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+/// An object that can be encoded and injected into an app using the launch environment.
+public protocol LaunchEnvironmentObject {
```

Comment:
Thoughts on a name like `LaunchEnvironmentEncodable`? Totally just a suggestion that came to mind and sounded bit more protocol-y